frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -666,6 +668,49 @@ export default function SurveyEdit(): JSX.Element {
                                                                 />
                                                             </div>
                                                         </LemonField.Pure>
+                                                        <LemonField.Pure
+                                                            label="Device Types"
+                                                            info="Select the device types to show the survey on"
```

Comment:
https://github.com/PostHog/posthog.com/pull/10529/
I'd say a bit more than that, and I'd use 'add' instead of 'select'.
I'd also add the possible values as described in the docs, probably the min. version of the SDK required as well

Reply:
just updated